test/unit/resources/postgres_ident_conf_test.rb

```diff
@@ -0,0 +1,28 @@
+# encoding: utf-8
+# copyright: 2017
+# author: Aaron Lippold, lippold@gmail.com
+# author: Rony Xavier, rx294@nyu.edu
+
+require 'helper'
+require 'inspec/resource'
+
+describe 'Inspec::Resources::PGIdentConf' do
+  describe 'PGIdentConf Paramaters' do
+    resource = load_resource('postgres_ident_conf')
+    it 'Verify postgres_ident_conf filtering by `system_username`'  do
```

Comment:
While these are fine tests, this isn't exactly what I was trying to convey with my earlier review. 🙂  Your tests are testing more the `.where` functionality of FilterTable than you are your parsing engine.

A better more-slimmed-down test might look like this:

```ruby
it 'parses the pg_ident.conf file correctly' do
  resource.map_name.must_equal ['omicron', 'ssl-test', 'pki-users']
  resource.system_username.must_equal ['bryanh', 'ann', 'robert']
  resource.pg_username.must_equal ['bryanh', 'ann', 'bob']
end
```

This eliminates any unnecessary calls to FilterTable and focuses solely on this resource's ability to parse your file and populate the FilterTable itself.

You also don't need the two `describe` statements here... just whittle it down to one and make it match the actual class name of the resource without quotes:

```ruby
describe Inspec::Resources::PostgresIdentConf do
  it 'parses the pg_ident.conf file correctly' do
    # stuff here
  end
end
```

Additional `describe` statements can always be added later to cover individual helper methods, etc.